test/regressions/fixtures/Autocomplete/SizeSmallValueOverflow.js

```diff
@@ -103,14 +93,12 @@ export default function Sizes() {
         defaultValue={[movies[0]]}
         disableClearable
         renderTags={(value, getTagProps) =>
```

Comment:
Looks like removing the `renderTags` prop also removes the key warning.

Reply:
This demo supports `multiple`. It needs `renderTags` prop to render small outlined chips.